project-base/.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,31 @@
+on: [push]
+name: "Docker build"
+jobs:
+    build-whole-application:
+        name: Build application and run standards checks and tests
+        runs-on: 'ubuntu-20.04'
```

Comment:
on Travis, we check also installation on Mac (actually it's Mac installation on Linux).
It's dropped intentionally?
What about investigating GitHub action on MacOS?

Reply:
I did not like that check honestly, it was weird - so I did not add it in Github actions.

I tried Github action with for MacOS, but there are problems with Docker in Github actions on MacOS see https://github.com/actions/virtual-environments/issues/17#issuecomment-614726536

I also found some tries to implement this, but none successful.

Reply:
it's possible to add the following to run docker on macOS, docker-sync is not installed. 
```
on: [push]
name: "MacOS test"
jobs:
    mac-os-test:
        name: Docker on MacOS test
        runs-on: macos-latest

        steps:
            -   name: GIT checkout branch - ${{ github.ref }}
                uses: actions/checkout@v2
                with:
                    ref: ${{ github.ref }}
            -   name: Setup Docker
                uses: docker-practice/actions-setup-docker@master
                with:
                    docker_channel: stable
            -   name: Test docker
                run: |
                    docker run --rm hello-world
                    docker-compose --version
                    docker-sync --version
```

I'm not sure whether it's a good idea to drop this test. 
It's testing installation with docker sync, and we need to do this manually instead, which is something, we wanted to avoid in the past.

Reply:
Hi, I tried few runs, last one without volumes and it is not ready to be used see: https://github.com/shopsys/project-base/runs/1861378295?check_suite_focus=true

Reply:
ok, let's do it without installation on mac